front-end/src/Components/Navigation/NavigationBar.tsx

```diff
@@ -0,0 +1,83 @@
+import { Route, Switch } from 'react-router-dom';
+import { action } from '@storybook/addon-actions';
+import styles from './NavigationBar.module.css';
+import {
+  Header,
+  HeaderContainer,
+  HeaderGlobalAction,
+  HeaderGlobalBar,
+  HeaderMenuButton,
+  HeaderMenuItem,
+  HeaderName,
+  HeaderNavigation,
+  HeaderSideNavItems,
+  SideNav,
+  SideNavItems,
+  SkipToContent,
+} from 'carbon-components-react';
+import { Search20, Notification20, AppSwitcher20, Add16 } from '@carbon/icons-react';
+const NavbarScroller = () => {
```

Comment:
Its best to have matching names with the file name here, so this should either be renamed `NavBar` and the file name to `NavBar.tsx` or have it as `NavigationBar` component. We should also have types for it since we already using typescript e.g 
```
interface NavigationBarProps { }

const NavigationBar: React.FC<NavigationBarProps> = () => {
  ... component implementation
}